docfiles/pxtweb/cookieCompliance.ts

```diff
@@ -261,10 +261,11 @@ namespace pxt {
     }
 
     /**
-     * Checks for winrt
+     * Checks for winrt and pxt-electron
      */
     function isNativeApp(): boolean {
-        return typeof Windows !== "undefined";
+        return typeof Windows !== "undefined" ||
+            !!(window as any).pxtElectron;
```

Comment:
check that window is defined in case it runs in node.

Reply:
done